BuildOpenCV.cmake

```diff
@@ -26,7 +26,14 @@ if (NOT EXISTS ${OPENCV_DOWNLOAD_DIR}/opencv-${OPENCV_VERSION}/CMakeLists.txt)
                 ${OPENCV_DOWNLOAD_DIR}/${OPENCV_VERSION}.zip
```

Comment:
Change it to `${OPENCV_DOWNLOAD_DIR}/opencv-${OPENCV_VERSION}.zip`. And don't forget to also change the `NOT EXISTS` check.